src/style.css

```diff
@@ -150,3 +148,207 @@ hr {
+  margin-top: 1rem;
+}
+
+.container {
+  width: 25rem;
+  height: auto;
+  background-color: rgb(54, 53, 53);
+  border-radius: 4rem;
+  padding: 0.5rem;
+}
+
+.search__box {
+  float: left;
+  width: 0;
+  height: 4rem;
+  background: none;
+  color: #f7f7f7;
+  font-size: 1.5rem;
+  border-radius: 2rem;
+  outline: none;
+  border: none;
+  position: relative;
+  opacity: 1;
+  transition: all 0.75s ease-in;
+  cursor: pointer;
+}
+
+.search__icon {
+  box-sizing: border-box;
+  font-size: 1.5rem;
+  display: inline-block;
+  justify-content: center;
+  align-items: center;
+  margin-left: 0.8rem;
+  margin-top: 0.8rem;
+  cursor: pointer;
+  position: absolute;
+  transition: all 0.25s ease-in;
+  border-radius: 50%;
+}
+
+.container:hover > .search__box {
+  width: 85%;
+  padding: 0 1rem;
+}
+
+.like-btn {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  column-gap: 10px;
+  min-width: 70px;
+  height: 60px;
+  font-size: 1.3rem;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  color: #bf0603;
+  margin-left: 20%;
+}
+
+.fa-heart {
+  color: rosybrown;
+  transform: scale(1.5);
+  transition: transform 0.5s;
+  fill: #bf0603;
+}
+
+.fa-heart:hover {
+  transform: scale(2);
+}
+
+.fa-solid {
+  animation: changeOpacity 2s cubic-bezier(0.075, 0.82, 0.165, 1);
+}
+
+@keyframes changeOpacity {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+/* Popup comments */
+.comment__popup {
+  width: 100vw;
+  background-color: #080213;
+  position: fixed;
+  height: 100%;
+  overflow: scroll;
+  z-index: 1000;
+  padding: 4% 20%;
+  display: none;
+}
+
+.popup__card {
+  width: 100%;
+  background-color: grey;
+  display: flex;
+  justify-content: center;
+}
+
+.popup__card img {
+  width: 80%;
+  height: 50rem;
+}
+
+.movie__data {
+  display: flex;
+  flex-direction: column;
+}
+
+.movie__header {
+  align-self: center;
+}
+
+.movie__data-details {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.comment__list {
+  list-style: none;
+}
+
+.comment__list li {
+  display: flex;
+  flex-direction: row;
+  background-color: #474649b5;
+  margin: 2rem 0;
+  padding: 1rem;
+  border-radius: 1rem;
+}
+
+.comment__list li span {
+  margin-right: 1rem;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+label {
+  width: 80%;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+}
+
+input {
+  width: 60%;
+  height: 2rem;
+  padding: 2rem;
+  color: #080213;
+}
+
+textarea {
+  width: 60%;
+  height: 10rem;
+  padding: 2rem;
+  margin-top: 2rem;
+  color: #080213;
+}
+
+input:focus,
+textarea:focus {
+  outline: none;
+}
+
+button[type="submit"] {
+  background-color: #fff;
+  align-self: flex-start;
+  color: #080213;
+  padding: 0.5rem 1rem;
+}
+
+.comment__popup-buton {
+  display: flex;
+  flex-direction: row-reverse;
+  width: 100%;
+  margin-bottom: 5px;
+}
+
+.hidde {
```

Comment:
This class is not exist in the HTML, its better to remove unused code and things like it